package.json

```diff
@@ -9,7 +9,7 @@
     "@testing-library/jest-dom": "^4.2.4",
     "@testing-library/react": "^9.5.0",
     "@testing-library/user-event": "^7.2.1",
-    "airtable": "^0.9.0",
+    "airtable": "^0.10.0",
```

Comment:
@Souvikns this thing made that Issue raised : #113 . I have fixed it now.

Reply:
@rarchitgupta please do not upgrade the package versions. It can seriously make applications crash.

Reply:
Sorry about that. Didn't mean to